fasm2bels/database/create_channels.py

```diff
@@ -437,3 +438,25 @@ def create_channels(db_root, part, connection_database):
         print("{}: Connections made".format(datetime.datetime.now()))
         count_sites_on_nodes(conn)
         print("{}: Counted sites".format(datetime.datetime.now()))
+
+
+def main():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        '--db-root',
+        required=True,
+        help="Path to prjxray database for given FASM file part.")
+    parser.add_argument(
+        '--part', required=True, help="Name of part being targeted.")
+    parser.add_argument(
+        '--connection-database',
+        required=True,
+        help="Path to SQLite3 database for the given part.")
```

Comment:
This points to an output file. I'd add the word "output" to the help string to clearly indicate that.

Reply:
Done